### PR TITLE
User model completed.

### PR DESCRIPTION
**Pull request by parthjdoshi**

### CODE REVIEW

File: users/models.py [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+
+
+# Creating a custom path for storing the user photos
+# Example : /MEDIA_ROOT/photos/1234567890/abc.jpg
+def path(instance, filename):
+    return 'photos/{0}/{1}'.format(instance.mobile, filename)
+
+
+# Creating a custom User model
+class User(AbstractUser):
```
Comment (doshivatsal7) [inline users/models.py]: Change the name to something other than User, it might conflict later with Django User

File: users/models.py [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+    # Manually check uniqueness during input
+    sap_id = models.CharField(max_length=15, unique=True)
+    # Fixed format of mobile like 1234567890
+    message = 'Phone number should be of 10 digits'
+    phone_regex = RegexValidator(regex=r'^[1-9][0-9]{9}$', message=message)
```
Comment (doshivatsal7) [inline users/models.py]: This isn't necessarily needed as long as we're sending the correct data from the front end

**Review by doshivatsal7 — CHANGES_REQUESTED** [review]
Please make the changes I've commented

File: users/models.py (hunk 1 above)
Comment (parthjdoshi) [reply users/models.py]: It shouldn't conflict since we have pointed settings.AUTH_USER_MODEL to this custom user, but I'll change it nevertheless.

**Comment by AvaisP** [comment]
@doshivatsal7 Please keep checking the PR once you request changes. See if they have been implemented here

**Comment by AvaisP** [comment]
@doshivatsal7 Can you review?

**Review by doshivatsal7 — APPROVED** [review]

**Pull request merged**